package.json

```diff
@@ -59,12 +59,12 @@
     "eslint-config-standard": "^14.1.0"
   },
   "peerDependencies": {
-    "eslint": ">=6.0.1",
+    "eslint": ">=6.2.2",
     "eslint-plugin-import": ">=2.18.0",
     "eslint-plugin-node": ">=9.1.0",
     "eslint-plugin-promise": ">=4.2.1",
     "eslint-plugin-standard": ">=4.0.0",
-    "@typescript-eslint/eslint-plugin": ">=2.0.0"
+    "@typescript-eslint/eslint-plugin": ">=2.3.3"
```

Comment:
I'm quite certain this should be the same version string as what we use as `devDependency`, because that's how we know which rules we can add.